packages/core/src/components/collapsible-list/collapsible-list.md

```diff
@@ -18,7 +18,9 @@ items using their [`IMenuItemProps`](#core/components/menu.menu-item).
 
 @interface ICollapsibleListProps
 
-@## Separators
+@## CSS
+
+### Separators
```

Comment:
`@### Separators` - it's not a full-fledged CSS API section (cuz the component is React only) but rather a usage note